### ISSUE TITLE
Spinner + Play Both Apear when no source

### ISSUE THREAD

**Issue by paolavness** [video-react/video-react#8]
Hello again,
Thank you for resolving the previous issue #7 so quickly. Really appreciated!

### Description
When src is '', LoadingSpinner and BigPlayButton. BigPlayButton is clickable and tries to play the src.

- version `#0.6.2`
- browser: `Chrome 55`

### Steps to reproduce issue
```
             this.props.videoURL = ''

             <Player className="video__el">
                  <BigPlayButton position="center" />
                  <source src={this.props.videoURL} />
                  <LoadingSpinner /> 
                  <ControlBar autoHide />
                  <Shortcut shortcuts={this.newShortcuts} />
                </Player>
```

### Steps to reproduce

1. Just use above code

### Expected behaviour
Expect only the LoadingSpinner to appear

### Actual behaviour
LoadingSpinner and BigPlayButton appear. BigPlayButton is clickable - on clicking ControlBar appears, with Play active (pause button), and LoadingSpinner disappears. When I click Pause, an error thrown:
`Uncaught (in promise) DOMException: The play() request was interrupted by a call to pause().`

**Comment by paolavness** [comment]
![screen shot 2017-01-20 at 8 39 52 am](https://cloud.githubusercontent.com/assets/20376932/22158946/862d7fda-def3-11e6-9e83-d360bf379f45.png)
![screen shot 2017-01-20 at 8 39 31 am](https://cloud.githubusercontent.com/assets/20376932/22158950/8b23c67a-def3-11e6-96af-67f6dd145574.png)

**Comment by JimLiu** [comment]
@paolavness 
Please test the code from master, I'm not sure if it's stable and fixed your issue.
If everything is ok, I'll publish to npm

**Comment by paolavness** [comment]
thanks again for a quick response.

I cloned from master and tried npm installing, then running, but quite a number of errors.

[intall-log.txt](https://github.com/video-react/video-react/files/720800/intall-log.txt)

I'll try get into my working environment but its going to take some time.

on a separate note, i played with the manager subscriptions today and they're working well. I've hooked into a react-redux app and am setting video.video.currentTime with in ComponentDidUpdate() to adjust the the current playing segment from via application state. Probably not the best way of doing things but working well!

**Comment by JimLiu** [comment]
Sorry I have no idea with the npm errors.

It has a new method `subscribeToStateChange` to do that from 0.6.x, no need to call `manager`.
There is a sample on how to use it: https://video-react.github.io/components/player/

**Comment by JimLiu** [comment]
@paolavness 
published 0.6.3 to npm, it fixed this issue. please have a check.
Thanks

**Comment by paolavness** [comment]
Awesome, thank you!
